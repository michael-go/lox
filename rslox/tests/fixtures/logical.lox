print 1 == 2;
print 1 == 1;
print (true and false);
print true or false;
print 1 > 2 or 2 > 2 or 3 > 2;
print 1 < 2 and 1 == 2;