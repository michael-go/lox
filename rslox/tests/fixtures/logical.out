# exit code: 0
# stdout:
false
true
false
true
true
false

# stderr:

